bedrock/firefox/templates/firefox/choose.html

```diff
@@ -0,0 +1,68 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Note to self: this page has previously been removed from bedrock, so looks like this may have been added back accidentally during a rebase. Need to check to make sure any other files haven't been reintroduced.